### DISCUSSION TITLE
Annoying bug in Raven Studio

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#14883] [General]
Could we fix the annoying bug in the Raven studio. 

If you do any query the last row of the returned results is always obscured by the "Your data has changed..".

![image](https://user-images.githubusercontent.com/4225690/188844558-1bc60903-8dc2-4a5b-808e-3f9afa1c834f.png)

**Comment by ppekrol** [comment]
@bobskigit are you working on a live system? We are informing you here that the results of a query might not be up to date, because some of the data has changed.

@ml054 can you take a look when this is displayed exactly?

**Reply by ml054** [reply]
@bobskigit thanks for letting us know. I moved issue to our official bug tracker. You can track progress here: https://issues.hibernatingrhinos.com/issue/RavenDB-19327/spurious-calls-in-collection-tracker 

Good news this issue is in progress. :)

**Reply by ml054** [reply]
@bobskigit I writing to let you know that issue was resolved and will be available in next stable build. 

Thanks for reporting this bug and happy hacking with RavenDB.

**Comment by bobskigit** [comment]
Yes I understand that the data is not complete hence the message but it shouldn’t obscure the last row of data.

**Comment by ml054** [comment]
@bobskigit this issue was resolved. Could you check if problem still persist after upgrade?

**Comment by bobskigit** [comment]
I will , thanks for fixing itOn 12 Oct 2022, at 07:51, Marcin Lewandowski ***@***.***> wrote:﻿
@bobskigit this issue was resolved. Could you check if problem still persist after upgrade?
